L/libcxxwrap_julia/build_tarballs.jl

```diff
@@ -55,7 +55,7 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-
+	Dependency("Julia_jll")
```

Comment:
This should be a [`BuildDependency`](https://juliapackaging.github.io/BinaryBuilder.jl/dev/reference/#BinaryBuilder.BuildDependency):
```suggestion
    BuildDependency("Julia_jll")
```
Need also to remove all Julia sources above